Identity credentials: IdentityRequestOptions was renamed DigitalCredentialRequestOptions

marcoscaceres opened the pull request:
#### f3067121f0b40ece169b4591d04a905460f27727
<pre>
Identity credentials: IdentityRequestOptions was renamed DigitalCredentialRequestOptions
<a href="https://bugs.webkit.org/show_bug.cgi?id=268135">https://bugs.webkit.org/show_bug.cgi?id=268135</a>
<a href="https://rdar.apple.com/problem/121636291">rdar://problem/121636291</a>

Reviewed by Chris Dumez and Aditya Keerthi.

Renames IdentityRequestOptions to DigitalCredentialRequestOptions throughout.

* Source/WebCore/CMakeLists.txt:
* Source/WebCore/DerivedSources-input.xcfilelist:
* Source/WebCore/DerivedSources-output.xcfilelist:
* Source/WebCore/DerivedSources.make:
* Source/WebCore/Modules/credentialmanagement/CredentialsContainer.cpp:
(WebCore::CredentialsContainer::requestIdentity):
* Source/WebCore/Modules/credentialmanagement/CredentialsContainer.h:
* Source/WebCore/Modules/credentialmanagement/CredentialsContainer.idl:
* Source/WebCore/Modules/credentialmanagement/DigitalCredentialRequestOptions.h: Renamed from Source/WebCore/Modules/credentialmanagement/IdentityRequestOptions.h.
* Source/WebCore/Modules/credentialmanagement/DigitalCredentialRequestOptions.idl: Renamed from Source/WebCore/Modules/credentialmanagement/IdentityRequestOptions.idl.
* Source/WebCore/Sources.txt:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:

Canonical link: <a href="https://commits.webkit.org/273596@main">https://commits.webkit.org/273596@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/dca7d855feafdf92264b290066f88dbb5e68694c

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/35776 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/14719 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/37922 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/38501 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/32194 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/36994 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/17154 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/11743 "Built successfully") | [💥 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/30956 "An unexpected error occured. Recent messages:Printed configuration; Skipping applying patch since patch_id isn't provided; Checked out pull request") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/36330 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/12460 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/31806 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/10921 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/10940 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/31972 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/39748 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/32524 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/32297 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/36873 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/11112 "Built successfully") | [❌ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/9004 "Found 1 new test failure: http/tests/inspector/dom/didFireEvent.html (failure)") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/34959 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/12851 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/8189 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/11617 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/11928 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

cdumez reviewed (approved):
LGTM

pxlcoder commented on Source/WebCore/Modules/credentialmanagement/DigitalCredentialRequestOptions.idl:
```diff
@@ -23,7 +23,7 @@
  */
 
-dictionary IdentityRequestOptions {
+dictionary DigitalCredentialRequestOptions {
```
Not new to this PR, but shouldn't this have an `EnabledBySetting=`?

cdumez replied:
Not needed since the dictionary type is actually not exposed to JS. It is never needed for dictionaries.

marcoscaceres replied:
Ah, that just made me realize that we only put `EnabledBySetting=` on the new method for the Digital Credential API. We didn't add it to the resulting interface object 😰.... adding that now.

marcoscaceres replied:
Sent the following to address the missing EnabledBySetting: 
https://github.com/WebKit/WebKit/pull/23289

marcoscaceres committed:
Identity credentials: IdentityRequestOptions was renamed DigitalCredentialRequestOptions
https://bugs.webkit.org/show_bug.cgi?id=268135
rdar://problem/121636291

Reviewed by Chris Dumez and Aditya Keerthi.

Renames IdentityRequestOptions to DigitalCredentialRequestOptions throughout.

* Source/WebCore/CMakeLists.txt:
* Source/WebCore/DerivedSources-input.xcfilelist:
* Source/WebCore/DerivedSources-output.xcfilelist:
* Source/WebCore/DerivedSources.make:
* Source/WebCore/Modules/credentialmanagement/CredentialsContainer.cpp:
(WebCore::CredentialsContainer::requestIdentity):
* Source/WebCore/Modules/credentialmanagement/CredentialsContainer.h:
* Source/WebCore/Modules/credentialmanagement/CredentialsContainer.idl:
* Source/WebCore/Modules/credentialmanagement/DigitalCredentialRequestOptions.h: Renamed from Source/WebCore/Modules/credentialmanagement/IdentityRequestOptions.h.
* Source/WebCore/Modules/credentialmanagement/DigitalCredentialRequestOptions.idl: Renamed from Source/WebCore/Modules/credentialmanagement/IdentityRequestOptions.idl.
* Source/WebCore/Sources.txt:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:

Canonical link: https://commits.webkit.org/273596@main
